### DISCUSSION TITLE
extended nameof operator

### DISCUSSION THREAD

**Discussion by ekisielinski** [dotnet/csharplang#2476] [Language Ideas] [answered]
Example:

`nameof(A, B, C)`

would be translated to:

`new string[] { nameof(A), nameof(B), nameof(C) };`

**Comment by HaloFour** [comment]
What would the use case for this be?  How often is it actually an issue that requires a language change?

**Comment by ekisielinski** [comment]
@HaloFour not often, so it would be only nice if C# team has extra money from manager to implement such small things :)

**Comment by gafter** [comment, marked as answer]
@ekisielinski If we had extra time to implement things we have a huge backlog of actually useful things.

**Comment by gafter** [comment]
See https://github.com/dotnet/csharplang/projects/4
